src/assistant.rs

```diff
@@ -63,25 +76,27 @@ impl OpenAIAssistant {
      * This function creates an Assistant and updates the ID of the OpenAIAssistant struct
      */
     async fn create_assistant(&mut self) -> Result<()> {
-        //Get the API url
-        let assistant_url = "https://api.openai.com/v1/assistants";
+        //Get the assistant API url
```

Comment:
Does this break assistant v1?

Reply:
This does not but the constructor certainly does. Let me address that.